server/handlers/handlers.go

```diff
@@ -388,6 +390,24 @@ func (ah *ActivityHandler) handleCreatedActivity(msg *clientmodels.Message, subs
 		}
 	}
 
+	// Update the last chat received at for the participants of DM/GM
+	if chat != nil && len(userIDs) > 0 {
+		// exclude the sender from the list of participants
+		participants := []string{}
```

Comment:
Propose sizing this to match the expected output:
```suggestion
		participants := make([]string{}, len(userIDs) - 1)
```

Reply:
I'm switching this to a capacity set `make([]string, 0, len(userIDs) - 1)`

I think there's risk in setting a strict length here because the source of this is an external API, and history gave me many reasons to be scared of setting hard expectations when data is external 😅

Reply:
Yes! Agree with `append` semantics, I should have suggested that :p